### PR TITLE
Update zh-CN.rc

### PR DESCRIPTION
**Pull request by robsean**
Update zh-CN.rc

### CODE REVIEW

File: base/applications/osk/lang/zh-CN.rc [hunk 1]
```diff
@@ -251,40 +252,40 @@ END
...
     BEGIN
-        MENUITEM "Enhanced Keyboard", IDM_ENHANCED_KB, CHECKED
-        MENUITEM "Standard Keyboard", IDM_STANDARD_KB
+        MENUITEM "增强型键盘", IDM_ENHANCED_KB, CHECKED
+        MENUITEM "标准型键盘", IDM_STANDARD_KB
         MENUITEM SEPARATOR
-        MENUITEM "Regular Layout", IDM_REG_LAYOUT, CHECKED, GRAYED
-        MENUITEM "Block Layout", IDM_BLOCK_LAYOUT, GRAYED
+        MENUITEM "常规布局", IDM_REG_LAYOUT, CHECKED, GRAYED
+        MENUITEM "大块布局", IDM_BLOCK_LAYOUT, GRAYED
```
Comment (kernelbin) [inline base/applications/osk/lang/zh-CN.rc:266]: 这里的Block Layout指的是下图中这种布局，而不是大块的。

Block Layout means layout displayed in the following picture, not something like a big chunk.

image

建议调整一下说法，比如翻译成「常规布局」和「块状布局」

I suggest adjusting the way to express, such as translate as 「常规布局」and「块状布局」

**Comment by Extravert-ir** [comment]
Please, update your name in commit message - it is broken in commit messages.
And in other PRs as well

**Comment by robsean** [comment]
> 
> 
> Please, update your name in commit message - it is broken in commit messages.
> And in other PRs as well

robsean

**Comment by robsean** [comment]
The last  name is zheng,the first name is jianping

**Comment by binarymaster** [comment]
Thanks! Will apply to this commit.

**Comment by binarymaster** [comment]
Merged now. :slightly_smiling_face: 

To change your name for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Zheng Jianping"
git config --global user.email robsean@126.com
```

Or just for the current repository:
```
git config user.name "Zheng Jianping"
git config user.email robsean@126.com
```

Also it would be nice if you specify your full name **Zheng Jianping** in [GitHub profile settings](https://github.com/settings/profile) (Name field).

Regards,

**Pull request merged**